README.md

```diff
@@ -165,6 +165,108 @@ Depending on the type of the joystick and the personal preference for operation,
+
+#### Grid specifications
+
+The grid used to define the wind field must be equidistant in x, respectively y-direction. The points in z-direction are distributed with upwards linearly increasing distance (spacing factors can be tuned as desired).
+
+#### Wind field text file format
+
+The text file contains information about the grid geometry as well as the wind values at each grid point. The data needed in the text file consists of:
+
+  1. Smallest x-coordinate of the grid `min_x`, of type integer, in [m].
+
+  2. Smallest y-coordinate of the grid `min_y`, of type integer, in [m].
+
+  3. Number of grid points in x-direction `n_x`, of type integer.
+
+  4. Number of grid points in y-direction `n_y`, of type integer.
+
+  5. Resolution in x-direction `res_x`, of type float, in [m].
+
+  6. Resolution in y-direction `res_y`, of type float, in [m].
+
+  7. Spacing factors in z-direction stored in `vertical_spacing_factors`, an (n_z)-dimensional array of float values between 0 for the lowest and 1 for the highest point.
+  8. The altitude of each grid point contained in the lower x-y plane, stored in `bottom_z`, an (n_x*n_y)-dimensional array of float values, in [m].
+  > **Note**: Element [0] is the grid corner point with the lowest x and y-coordinates, and the array is filled counting up in x-direction first, then in y-direction (such that a point with indices i,j corresponds to the (i+j*n_x)th element of the array).
+
+  9. Similarly, the altitude of each grid point contained in the upper x-y plane, stored in `top_z`, an (n_x*n_y)-dimensional array of float values, in [m].
+
+  10. The speed of the wind in x-direction for each grid point, stored in `u`, an (n_x*n_y*n_z)-dimensional array of float values, in [m/s].
+  > **Note**: Element [0] is the grid corner point with the lowest x,y and z-coordinates, and the array is filled counting up in x-direction first, then in y-direction and finally in z-direction (such that a point with indices i,j,k corresponds to the (i + j*n_x + k*n_x*n_y)th element of the array).
+
+  11. Similarly, the speed of the wind in y-direction stored in the array `v`, and in z-direction stored in `w`.
+
+The order in which the data is saved in the text file is not relevant, but the format must comply with the following requirements:
+
+  1. In the first line of the file, the name of one of the 12 needed data followed directly by a semicolon (e.g., `vertical_spacing_factors:`)
+
+  2. In the following line, the corresponding data. If multiple values are needed, they must be separated by a space (e.g., `0.0 0.025641 0.051282 0.076923 ...`)
+
+  3. The rest of the data follows the same format.
+
+In the case of the hemicylindrical world, the beginning of the text file describing a divergent-free wind field looks as follows:
```

Comment:
Alright. Small question though: the hemicyl world does not exist in the main repo, since I created it... Should I include it in the next PR?